tests/libpod/render.go

```diff
@@ -198,3 +209,32 @@ func RenderTargetcliPod(name, disksPVC string) *v1.Pod {
 		},
 	}
 }
+
+func AddServiceBackHoleForKubernetesApiToPod(pods *v1.PodList, virtCli kubecli.KubevirtClient, componentName string) {
+	serviceBlackHoleForKubernetesApiToPod(pods, virtCli, componentName, BlackHoleActionAdd)
+}
+
+func DeleteServiceBlackHoleForKubernetesApiToPod(pods *v1.PodList, virtCli kubecli.KubevirtClient, componentName string) {
```

Comment:
```suggestion
func DeleteKubernetesApiBlackhole(pods *v1.PodList, virtCli kubecli.KubevirtClient, componentName string) {
```

Reply:
Call it `containerName` instead of `componentName`?

Reply:
Done.